### ISSUE TITLE
On the Application of Efficient Neural Mapping to Real-Time Indoor Localisation for Unmanned Ground Vehicles. (arXiv:2211.04718v2 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#29179]
On the Application of Efficient Neural Mapping to Real-Time Indoor Localisation for Unmanned Ground Vehicles. (arXiv:2211.04718v2 [cs.RO] UPDATED)<br>
https://ift.tt/BEIDdMJ<br>
<p>Global localisation from visual data is a challenging problem applicable to
many robotics domains. Prior works have shown that neural networks can be
trained to map images of an environment to absolute camera pose within that
environment, learning an implicit neural mapping in the process. In this work
we evaluate the applicability of such an approach to real-world robotics
scenarios, demonstrating that by constraining the problem to 2-dimensions and
significantly increasing the quantity of training data, a compact model capable
of real-time inference on embedded platforms can be used to achieve
localisation accuracy of several centimetres. We deploy our trained model
onboard a UGV platform, demonstrating its effectiveness in a waypoint
navigation task, wherein it is able to localise with a mean accuracy of 9cm at
a rate of 6fps running on the UGV onboard CPU, 35fps on an embedded GPU, or
220fps on a desktop GPU. Along with this work we will release a novel
localisation dataset comprising simulated and real environments, each with
training samples numbering in the tens of thousands.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
